owasp-zap/README.md

```diff
@@ -0,0 +1,39 @@
+# OWASP Zed Attack Proxy
+
+[ZAP](https://www.zaproxy.org/) is a security pen testing tool, which this project uses for active scanning for
+vulnerabilities. To manually run a scan similar to the ones integrated into [our GitHub Actions](../.github/workflows),
+run `./manual-scan.sh` from a command line. For full details on running the scan locally, see [Manual run ](#manual-run). Below is a brief explanation of the pieces of this integration and what they are for.
+
+# Authenticator
+
+The [authenticator](./authenticator) directory is a simple NodeJS script that leverages
+[aws-amplify](https://www.npmjs.com/package/aws-amplify) to quickly log a test user in and acquire an token for
+authentication. This script requires that the Cognit UserPool Client be configured to enable the SRP authentication
+flow, which can only be activated in pre-production environments.
+
+# Data
+
+The [data](./data) folder contains configuration and automation data files that are used to control ZAP's scan. This
+includes an HttpSender script and an automation YML file. The HTTPSender script will tell ZAP to add the token we acquired from the [authenticator](#authenticator) script into the `Authorization` header for every 'in scope' request, headed
+to the UI or API. The YML file defines what jobs are to be run as part of an automated scan.
+
```

Comment:
Do we need a section here for how to update to the ZAP data in the event that more API endpoints are added or other resources need to be tested?

Reply:
No need. This ZAP automation includes ingesting our official API Spec via latest-oas30.json, so it will always have the latest API definition as part of its input. Also, we're going to add similar automation for practitioner users as a second scan, probably, next.